src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,7 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomColor().toString());
```

Comment:
toString will be called behind the scenes by default, create a for loop - we should print more than one ball